docs/configuration.md

```diff
@@ -0,0 +1,87 @@
+<!--
+    Copyright 2019 VMware, Inc.
+    SPDX-License-Identifier: BSD-2-Clause
+-->
+
+# Precaution configuration
+
+Precaution is using a variety of tools under the hood without exposing
+implementation details and complexity of the underlying tools to the user.
+
+Concretely, this means that Precaution ignores any tool specific configuration
+files which may exist in the project's repository and performs scans based solely
+on Precaution specific configuration.
+
+Precaution provides you with the option to use optional `.precaution.yaml` file.
```

Comment:
"option to use optional" sounds awkward because you're effectively saying twice that something is optional